### PR TITLE
Add local up install doc

### PR DESCRIPTION
**Pull request by JimmyYang20**

### CODE REVIEW

File: docs/setup/local-up.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### Deploy local Sedna clusters
+
+Once you have docker running you can create local Sedna clusters with:
```
Comment (llhuii) [inline docs/setup/local-up.md:3]: Add a description such as it uses kind for creating a k8s cluster, and a link for kind

File: docs/setup/local-up.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+### Deploy local Sedna clusters
+
+Once you have docker running you can create local Sedna clusters with:
+```
+bash scripts/local-up.sh
+```
+You can watch clusters with:
```
Comment (llhuii) [inline docs/setup/local-up.md:7]:
```suggestion
This script will boot a k8s cluster with one master node and a edge node, you can see them using `kubectl`:

```

Comment (JimmyYang20) [reply docs/setup/local-up.md:7]: k8s -> KubeEdge

File: docs/setup/local-up.md (hunk 1 above)
Comment (JimmyYang20) [reply docs/setup/local-up.md]: ok

File: docs/setup/local-up.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+### Deploy local Sedna clusters
+
+Once you have docker running you can create a local Sedna cluster with:
+```
+bash scripts/local-up.sh
+```
+This script uses [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a 
+local k8s cluster, and boot a KubeEdge cluster with one master node and one edge node.
```
Comment (llhuii) [inline docs/setup/local-up.md:8]:
```suggestion
local k8s cluster with one master node, and boots one edge node by running KubeEdge.
```

File: docs/setup/local-up.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+### Deploy local Sedna clusters
+
+Once you have docker running you can create a local Sedna cluster with:
```
Comment (llhuii) [inline docs/setup/local-up.md:3]:
```suggestion
Once you have docker running, you can create a local Sedna cluster with:

```

**Pull request closed without merging**